app/components/annotation/Answer.js

```diff
@@ -1,3 +1,4 @@
+// TODO: marked for deletion: topic picking view doesn't need Q&A
```

Comment:
The topic picker view should probably still have a (somewhat pruned) tree
displaying the information that will be sought in the quiz interface.
Remember, the topic will be described AND this tree will be there for
perusal, while the user (of the 1st interface) highlights topic text in
some broader tract (e.g. a news article).



On Mon, Feb 15, 2016 at 10:56 AM, Tyler Burton <notifications@github.com>
wrote:

> In app/components/annotation/Answer.js
> <https://github.com/Goodly/annotator-content-analysis/pull/20#discussion_r52933060>
> :
>
> > @@ -1,3 +1,4 @@
> > +// TODO: marked for deletion: topic picking view doesn't need Q&A
>
> ok @nickbadams <https://github.com/nickbadams> if this really is changing
> that drastically then new UX will be critical. otherwise this will take
> much longer to achieve.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Goodly/annotator-content-analysis/pull/20/files#r52933060>
> .
>



-- 

*Nick Adams, Ph.D.Data Science Fellow, *

*Berkeley Institute for Data Science*
*University of California, Berkeley*